src/quantuminspire/projectq/backend_qx.py

```diff
@@ -305,8 +434,9 @@ def _map_state_to_bit_string(self, state: int, qureg: List[Qubit]) -> str:
 
         for i in range(len(qureg)):
```

Comment:
why not `for qubit in qureg:`?

Reply:
Ahh, because you did not change this line, that's why

Reply:
Yeah, but now I changed it 👍